codalab/server/rest_server.py

```diff
@@ -135,6 +136,9 @@ def wrapper(*args, **kwargs):
                 code, message = exception_to_http_error(e)
                 if code == INTERNAL_SERVER_ERROR:
                     traceback.print_exc()
+                    # Internal error details should not be echoed to client
+                    message = 'Server encountered unexpected error while ' \
```

Comment:
Sorry, it slipped in when I was doing the massive rebasing to split the original PR.